roles/signals.py

```diff
@@ -0,0 +1,18 @@
+"""
+Signals handlers
+"""
+from __future__ import unicode_literals
+
+from django.db.models.signals import post_migrate
+from django.dispatch import receiver
+
+from roles.utils import sync_groups_permissions
+
+
+# pylint: disable=unused-argument
+@receiver(post_migrate)
+def my_callback(sender, **kwargs):
```

Comment:
It should be easy.  `sender` is an `AppConfig` instance, so you just need to add:
```python
if sender.name == 'learningresources':
...
```
wrapper around it: https://docs.djangoproject.com/en/1.8/ref/signals/#post-migrate and https://docs.djangoproject.com/en/1.8/ref/applications/#configurable-attributes for reference.